src/api/HttpLink.js

```diff
@@ -1,12 +1,20 @@
 import { BatchHttpLink } from '@apollo/client/link/batch-http/index';
 import { HttpLink } from '@apollo/client/link/http/index';
+import { split } from '@apollo/client/core/index';
+
+const cachableQueryOperationNames = [
+	'configSetting',
```

Comment:
What do you think about setting these via environment variables? Or does temporary nature of this addition negate the need?

Reply:
Operation names correlate to the named queries/mutaions in our .graphql files so I think these should remain here as they will relate to queries that are safe to cache.

I think it will take a pretty extensive review and configuration effort before we could run all queries through their graph. There are a lot of things that can't or shouldn't be cached and controlling that at the type layer isn't granular enough. As I reviewed more the general tab I see that we wouldn't want to cache much of that for very long and definitiely not wholesale as the config is currently set. We'll need to go through all of the queries and their fields and detail what the appropriate rules for them.

Reply:
That makes sense. I just meant setting this list as a comma separated env var (like `CACHEABLE_OPERATION_NAMES: operationOne,operationTwo` and `join` it in the code) to make additions/subtractions easy to test. Whatever works best though.